openapi/apis/core.json

```diff
@@ -12876,10 +12902,10 @@
                 }
               },
               "properties": {
-                "transactions": {
-                  "type": "array",
-                  "items": {
-                    "$ref": "#/components/schemas/Transaction"
+                "": {
```

Comment:
Don't use empty quotes as key. Give name of the property

Reply:
Changed